lib/puppet/type/apt_key.rb

```diff
@@ -59,9 +59,9 @@
 
   newparam(:server) do
     desc 'The key server to fetch the key from based on the ID. It can either be a domain name or url.'
-    defaultto :'keyserver.ubuntu.com'
+    defaultto :'hkps://keyserver.ubuntu.com'
```

Comment:
changing the default behaviour is dangerous. This would make is an X.y.z release. Was this changed due to security implications ?

Reply:
yes, change to hkps is a security implementation.